snapcraft/extensions/gnome.py

```diff
@@ -291,6 +322,19 @@ def get_parts_snippet(self) -> Dict[str, Any]:
         """
         source = get_extensions_data_dir() / "desktop" / "command-chain"
 
+        gpu_parts = {}
+        if self.yaml_data["base"] == "core24":
+            gpu_parts["gpu-2404"] = {
+                "after": list(self.yaml_data["parts"].keys()),
+                "source": "https://github.com/canonical/gpu-snap.git",
+                "plugin": "dump",
+                "override-prime": (
+                    "craftctl default\n"
+                    "${CRAFT_PART_SRC}/bin/gpu-2404-cleanup mesa-2404"
+                ),
+                "prime": ["bin/gpu-2404-wrapper"],
+            }
+
```

Comment:
Since you don't want the cleanup, and rely on linter instead, this will likely go away, and [the wrapper](https://github.com/canonical/gpu-snap/blob/main/bin/gpu-2404-wrapper) should probably be vendored in.